coalib/bearlib/__init__.py

```diff
@@ -38,6 +38,20 @@ def deprecate_settings(**depr_args):
      >>> func(new='coala!')
      coala!
 
+     This example represents the case where the old and new settings are
+     provided to the function.
+
+     >>> @deprecate_settings(new='old')
+     ... def run(new):
+     ...     print(new)
+     >>> run(old="Hello!", new='coala is always written with lowercase `c`.')
+     The setting `old` is deprecated. Please use `new` instead.
+     The value of `old` and `new` are conflicting. `new` will be used instead.
+     coala is always written with lowercase `c`.
+     >>> run(old='Hello!', new='Hello!')
+     The setting `old` is deprecated. Please use `new` instead.
+     Hello!
```

Comment:
can you give old an old value and new a new value and make sure that the new one's used?